src/lib/ui/svelte/customPageSvelte.js

```diff
@@ -0,0 +1,126 @@
+module.exports = `
+<script>
+  import heroMinaLogo from '$lib/assets/HeroMinaLogo_gray.svg'
+  import arrowRightSmall from '$lib/assets/arrow-right-small.svg'   
+  import { onMount } from "svelte";
+  import { isReady, Mina, PublicKey } from 'snarkyjs';
+	
+
+  onMount(async () => {
+    await isReady;  
+
+    const { Add } = await import('../../../contracts/build/src/')
+    // Update this to use the address (public key) for your zkApp account
+    // To try it out, you can try this address for an example "Add" smart contract that we've deployed to
+    // Berkeley Testnet B62qk33E2Kg31SBSpAGUDkjzDGHk9LFt2N1vKFyHy3ezXdd6eiJeAqa
+    const zkAppAddress = ''
+    // This should be removed once the zkAppAddress is updated.
+    if (!zkAppAddress) {
+      console.error(
+        'The following error is caused because the zkAppAddress has an empty string as the public key. Update the zkAppAddress with the public key for your zkApp account, or try this address for an example "Add" smart contract that we deployed to Berkeley Testnet: B62qqkb7hD1We6gEfrcqosKt9C398VLp1WXeTo1i9boPoqF7B1LxHg4',
+      );
+    }
+    const zkApp = new Add(PublicKey.fromBase58(zkAppAddress))
+  });
+
+</script>
+<svelte:head>
+  <title>zkApp CLI</title>
+</svelte:head>
+<main class="main">
+  <div class="center">
+    <img
```

Comment:
What do you think about making the logo a clickable link to minaprotocol.com? This could be a nice touch to help drive some traffic to the website. Let me know your thoughts!

Reply:
Great idea!

Reply:
Considering that the zkapp docs are already linked elsewhere, it might be more beneficial to have the logo link to the minaprotocol domain instead.